src/validations/request.validation.js

```diff
@@ -50,4 +51,37 @@ async function requestValidation(req, res, next) {
   next();
 }
 
-export default requestValidation;
+export const validateChangingRequestStatus = async (req, res, next) => {
+  const schema = Joi.object({
+    requestId: Joi.number().required(),
+    status: Joi.string().valid('approved', 'rejected').trim()
+      .messages({
+        'any.only': 'status" must be one of approved or rejected'
+      })
+      .required()
+  }).options({ abortEarly: false });
+
+  const results = schema.validate({ ...req.params, ...req.body });
+  if (results.error) {
+    const errorMessages = [];
+    results.error.details.forEach((error) => {
```

Comment:
How many times more do I have to give feedback on the use of .map instead of .forEach?

Reply:
I'm sorry I missed it. it won't happen again. Thanks.